app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheet.kt

```diff
@@ -134,7 +165,10 @@ class ScheduleAppointmentSheet : BottomSheetActivity(), ScheduleAppointmentUi, S
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.sheet_schedule_appointment)
+
+    binding = SheetScheduleAppointmentBinding.inflate(layoutInflater)
```

Comment:
Same as the previous commit comment

Reply:
Same comment as above. We don't need to do that for Activities